### ISSUE TITLE
Finer volume control

### ISSUE THREAD

**Issue by swedneck** [ashutoshgngwr/noice#660]
The current volume sliders don't give a lot of control, there are a low number of steps and a simple percentage isn't great for adjusting low volumes.

It would be nice to increase the amount of steps and give an option for logarithmic sliders to give more fine grained control over volume.

**Comment by ashutoshgngwr** [comment]
I did some math. The current system maps volume levels [0, 20] to [0, 1] linearly. I scaled the current system to [0, 25] ([1, 25] in logarithmic case) and plotted its mapping to [0, 1] using logarithmic, quadratic and cubic functions. Shouldn't the control be a quadratic or cubic function instead of logarithmic? Because according to my experience with the current version of Noice, the sound volumes decline very rapidly towards the lower range while the upper range barely has any effect.

![image](https://user-images.githubusercontent.com/7106123/121028635-9d829e80-c7c5-11eb-9c18-5bab7bf7f2cc.png)
